04/ceasar_cipher_final.py

```diff
@@ -0,0 +1,24 @@
+#Caesar cipher
+
+plaintext = input ("enter text you want to cipher: ")
+key = input ("enter cipher key: ")
+
+while key.isdigit()== False or (int(key) <= 0):
```

Comment:
Metoda isdigit vrací boolean - True nebo False. Není třeba porovnávat, jestli False == False, stačí napsat 

`while key.isdigit() or (int(key) <= 0):` :slightly_smiling_face:

Reply:
A nebylo by to potom blbě? Jediná situace, kdy by pak neproběhl ten while cyklus by byla, kdyby to bylo False a False -> input by nebylo číslo a bylo by větší než 0. Já potřebuju aby While neproběhlo, když input JE DIGIT a >0. Neexistuje nějaká metoda .isntdigit()? :)

Reply:
Jo, promiň :D Je tam potřeba NOT.
`while not key.isdigit() or (int(key) <= 0):` :slightly_smiling_face: